### PR TITLE
header

### PR DESCRIPTION
**Pull request by marcushohlbein**

### CODE REVIEW

File: client/src/components/Header/Header.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import logo from '../../assets/sneaker24-logo.png'
+
+export default function Header({ text }) {
```
Comment (macfrei) [inline client/src/components/Header/Header.js]: If you don't use the prop `text`, why do you have it? :)

**Review by macfrei — COMMENTED** [review]
Short and sweet 😸

**Review by bwnstck — APPROVED** [review]
looks good :)

File: client/src/components/Header/Header.js (hunk 1 above)
Comment (marcushohlbein) [reply client/src/components/Header/Header.js]: That's a good question. Don't know how this prop sneaked in 😂

**Pull request merged**